libs/ui/src/lib/header/header.component.html

```diff
@@ -1,10 +1,8 @@
 <mat-toolbar color="primary">
   <mat-toolbar-row>
-    <a mat-button routerLink="home"> Home </a>
+    <a mat-button routerLink="landing"> Home </a>
     <div>
-      <a mat-button routerLink="faq"> FAQ </a>
       <a mat-button routerLink="marketplace"> Marketplace </a>
-      <a mat-button routerLink="contact"> Contact Us </a>
       <a mat-button routerLink="my-nft"> My NFTs </a>
```

Comment:
This isn't navigating to the correct page

Reply:
I need to check this.

Reply:
Changing this to "my-tokens"